### DISCUSSION TITLE
Questions

### DISCUSSION THREAD

**Discussion by DarthVan** [SGGames/DragonBones-Pixi#1] [Ideas]
Hi man! Is this project already can play simple sceleton animations in pixi v8 ?
You can’t imagine how many people want an alternative to spine, even if only some of the features can be implemented. I hope everything works out for you, and thank you for your work! 👍

**Comment by atomixnmc** [comment]
Hi @DarthVan, thank you for your great compliment. 
Yes, skeleton animations works already, take a look at the examples: https://github.com/SGGames/DragonBones-Pixi/tree/master/src/adapters/pixi/demo

We are working on a **free and open source** version of Next-gen Animation Editor call AnigoStudio and as soon as we have the working prototype we gonna open source it too.

**Reply by DarthVan** [reply]
This is great news, it would be awesome! Thanks a lot!
